### PR TITLE
Update README.md to include relevant links

### PR DESCRIPTION
**Pull request by MitoG**
Added relevant links to the official Repo, Website, Documentation and Discord.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,21 @@
...
+ 
+**Official [Website](https://ub3r-b0t.com/)**
+
+**Official [Documentation](https://ub3r-b0t.com/docs/articles/intro.html)**
+
+**Support [Discord](https://discord.gg/0t6waxpRf4xvyNHj)**
+
+----
+
+### Articles
```
Comment (moiph) [inline README.md]: I'd rather leave out the articles section and just leave the top level Docs link above

Comment (MitoG) [reply README.md]: that's fine, was just an idea.

Comment (moiph) [reply README.md]: Once you update to remove the articles section I'll get the rest merged

Comment (MitoG) [reply README.md]: README has been updated, the links have been removed.

**Review by moiph — APPROVED** [review]

**Pull request merged**